src/components/Button/index.jsx

```diff
@@ -0,0 +1,13 @@
+import styles from './Button.module.scss';
+
+export const Button = ({id, type, text, ...functions}) => {
```

Comment:
```suggestion
export const Button = ({id, type, text, onClick, ...props}) => {
```
onClickは必ず必要だと思うので明記しましょう。
あとfunctionsではなく、propsのほうが良さそうです。